### DISCUSSION TITLE
Loading .bgeo toHoudini error

### DISCUSSION THREAD

**Discussion by Perpetual3134** [InteractiveComputerGraphics/SPlisHSPlasH#246] [Q&A]
Hi there,

currently trying to load .bgeo files created with SPlisHSPlasH in Houdini, but getting this error:

Error | Unable to read file ".../partio/ParticleData_Fluid_1.bgeo".

Error | Expected array object (near byte offset 11, line 1, column 12)Backend IO

Does anyone have an idea what to do?

**Comment by janbender** [comment]
Hi, 

The problem might be that we use an older version of the bgeo file format. The reason for this is the surface reconstruction tool we use cannot handle the new format. 

You can try to update the partio library in SPlisHSPlasH. I think this might help.

**Reply by Perpetual3134** [reply]
Thanks!
Unfortunately I have little experience with this building stuff...

I replaced the partio files in the "extern" - folder at my local splishsplash with the new ones from the partio-github

Cmake throws this Error:

CMake Error at extern/partio/src/lib/CMakeLists.txt:44 (set_target_properties):
  set_target_properties called with incorrect number of arguments.

i tried to use the "original" CMakeLists.txt from the Splishsplash files, but that will not work too...

Do you have any tip for me what to do?

**Comment by Perpetual3134** [comment]
Or could you say which houdini version can use the generated .bgeo files from splishsplash?

**Comment by janbender** [comment]
I is hard to say what went wrong with the CMake file when I cannot debug it here. Since we don't use Houdini, I don't know which version can load the files. Alternatively, SPlisHSPlasH provides a command line tool called partio2vtk which converts the particle data to VTK files. Maybe you can read them. At least there seem to be some plugins for Houdini for VTK files. 

Hope that helps.
